nautobot/extras/models/jobs.py

```diff
@@ -170,6 +171,12 @@ class Job(PrimaryModel):
         help_text="Maximum runtime in seconds before the job will be forcibly terminated."
         "<br>Set to 0 to use Nautobot system default",
     )
+    worker_queues = JSONArrayField(
+        base_field=models.CharField(blank=True, null=True, max_length=100),
```

Comment:
if we remove `null=True` here. Would that resolve the inconsistency between `""` and `null` in `worker_queues`?